src/main/java/omero/gateway/facility/SearchFacility.java

```diff
@@ -112,7 +112,7 @@ public SearchResultCollection search(SecurityContext ctx, SearchParameters conte
                 // set general parameters
                 service.clearQueries();
                 service.setAllowLeadingWildcard(true);
-                service.setCaseSensitive(false);
+                service.setCaseSentivice(false);
```

Comment:
We'll need a reminder to roll this back. Alternatively, we go ahead and release m5 across the board and that is what the first omero-gateway consumes.

Reply:
I was going to add a comment to the PR so it gets updated when m5 is out